tock/api/views.py

```diff
@@ -4,44 +4,94 @@
 from django.db.models import Count, Sum
 from decimal import Decimal
 
-from django.contrib.auth.models import User, Group
-from hours.models import ReportingPeriod, Timecard, TimecardObject
-from projects.models import Project, Agency, AccountingCode
+from django.contrib.auth.models import User
+from projects.models import Project
+from hours.models import TimecardObject
 
 from rest_framework import serializers, generics, pagination, renderers
 
-from rest_framework.renderers import JSONRenderer
-from .renderers import PaginatedCSVRenderer
-
 import csv
+from .renderers import PaginatedCSVRenderer, stream_csv
 
 class StandardResultsSetPagination(pagination.PageNumberPagination):
+    """
+    This is a standard results set paginator for all API view classes
+    that need pagination.
+    """
     page_size = 100
     page_size_query_param = 'page_size'
     max_page_size = 500
 
+
+# Serializers for different models
+
 class ProjectSerializer(serializers.ModelSerializer):
-    billable = serializers.BooleanField(source='accounting_code.billable')
     class Meta:
         model = Project
-        fields = ('id', 'name', 'description', 'billable',)
+        fields = (
+            'id',
+            'name',
+            'description',
+            'billable',
+        )
+    billable = serializers.BooleanField(source='accounting_code.billable')
 
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = User
-        fields = ('id', 'username', 'first_name', 'last_name',)
+        fields = (
+            'id',
+            'username',
+            'first_name',
+            'last_name',
+        )
 
 class TimecardSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = TimecardObject
+        fields = (
```

Comment:
Do we need to list fields in `Meta` after declaring them as descriptors on `TimecardSerializer`? If the only reason to declare fields here is to add `hours_spent`, we could accomplish the same thing with less code by adding `hours_spent` as a field descriptor on the schema class.